.gitignore

```diff
@@ -32,3 +32,6 @@ site/
 /.envrc
 reference-tests/src/test/java/net/consensys/linea/generated/*
 !reference-tests/src/test/java/net/consensys/linea/generated/.gitkeep
+arithmetization/bin/
+acceptance-tests/bin/
+reference-tests/bin/
```

Comment:
Do these folders exist on your computer?

Reply:
yes I have these, not sure which gradle task creates them